### ISSUE TITLE
Possibility to use forces directly with hoomd-tf

### ISSUE THREAD

**Issue by moradza** [ur-whitelab/hoomd-tf#103]
I am new to hoomd-blue and hoomd-tf, and I am learning both at the same time,
I have a tensorflow mdoel that I want to turn into a graphe. My model gets the position of  atoms within a cut-off and give back the forces and not the potential. Is it possible to do this with hoomd-tf? For example a modification like below works or not?
Let's say the force is defined as r/norm(r)2, r is the vector between atom i and j,
f_vec = norm(f) * e_r, norm(f) = 1/norm(r)

graph = htf.graph_builder(64) # max neighbors = 64
rinv = graph.nlist_rinv 
f = my_tf_model(rinv)
forces = tf.reduce_sum( tf.multiply(f *r, rinv ), axis=-1)# there is mismatch between shape but lets say we solve it, (r is a vector )
graph.save('my_model', forces)

**Comment by whitead** [comment]
Yes, this is part of what hoomd-tf is for: to use other models for computing force. You don't need to have the potential, but the `forces` tensor should be shape Nx4. The final dimension contains x,y,z,E where E is assumed to be per-atom potential energy. You can just put 0s in there or nothing, it is not used. It is just there for consistency because if you get forces from hoomd-blue, the E's are populated.

**Comment by whitead** [comment]
Closing this for now. Let us know if you still need help
